teamcity: failed tests on master: testrace/TestEndTransactionDirectGCFailure

The following tests appear to have failed:

[#278284](https://teamcity.cockroachdb.com/viewLog.html?buildId=278284):

```
--- FAIL: testrace/TestEndTransactionDirectGCFailure (0.000s)
Test ended in panic.

------- Stdout: -------
I170622 03:31:29.610374 11882 storage/store.go:1262  [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I170622 03:31:29.627691 11948 storage/replica_command.go:188  [s1,r2/1:{a -/Max}] test injecting error: boom
E170622 03:31:29.629113 11947 util/log/crash_reporting.go:82  a panic has occurred!
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x170532e]

goroutine 11947 [running]:
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Recover(0xc420674000, 0x7fa770ec2b80, 0xc420014e40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:180 +0x10d
panic(0x215b260, 0x321e230)
	/usr/local/go/src/runtime/panic.go:489 +0x2f0
github.com/cockroachdb/cockroach/pkg/storage.(*Store).Send.func1(0xc4207dd390, 0xc4207dd400, 0x14ca54185a7bc5d4, 0x0, 0xc4207dd3f8, 0xc420068700)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:2478 +0x53e
runtime.Goexit()
	/usr/local/go/src/runtime/panic.go:366 +0x15c
testing.(*common).FailNow(0xc42099c1a0)
	/usr/local/go/src/testing/testing.go:486 +0x59
testing.(*common).Fatalf(0xc42099c1a0, 0x233c8e0, 0x19, 0xc4207f74c8, 0x1, 0x1)
	/usr/local/go/src/testing/testing.go:530 +0x95
github.com/cockroachdb/cockroach/pkg/storage.TestEndTransactionDirectGCFailure.func1(0x7fa770ec6e78, 0xc4202f4570, 0xc4202d6040, 0x8, 0x0, 0x1, 0x3263ca0, 0xc42009c070, 0x14ca54185a7baf90, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica_test.go:3984 +0x14b
github.com/cockroachdb/cockroach/pkg/storage.evaluateCommand(0x7fa770ec6e78, 0xc4202f4570, 0xc4202d6040, 0x8, 0x0, 0x7fa770f19910, 0xc4203b6370, 0xc4204b8a80, 0xc422cec000, 0xc42009c770, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica_command.go:187 +0xd27
github.com/cockroachdb/cockroach/pkg/storage.evaluateBatch(0x7fa770ec6e78, 0xc4202f4570, 0xc4202d6040, 0x8, 0x7fa770f19910, 0xc4203b6370, 0xc4204b8a80, 0xc422cec000, 0xc42009c770, 0x14ca54185a7baf90, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:4616 +0x3c8
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).evaluateTxnWriteBatch(0xc4204b8a80, 0x7fa770ec6e78, 0xc4202f4570, 0xc4202d6040, 0x8, 0x14ca54185a7baf90, 0x0, 0x100000001, 0x1, 0x1, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:4429 +0x2cb
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).evaluateProposalInner(0xc4204b8a80, 0x7fa770ec6e78, 0xc4202f4570, 0xc4202d6040, 0x8, 0x14ca54185a7baf90, 0x0, 0x100000001, 0x1, 0x1, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:4268 +0x137
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).evaluateProposal(0xc4204b8a80, 0x7fa770ec6e78, 0xc4202f4570, 0xc4202d6040, 0x8, 0x14ca54185a7baf90, 0x0, 0x100000001, 0x1, 0x1, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:2606 +0x131
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).requestToProposal(0xc4204b8a80, 0x7fa770ec6e78, 0xc4202f4570, 0xc4202d6040, 0x8, 0x14ca54185a7baf90, 0x0, 0x100000001, 0x1, 0x1, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:2565 +0x2f1
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).propose(0xc4204b8a80, 0x7fa770ec6e78, 0xc4202f4570, 0x0, 0x0, 0x14ca54188f1bdf08, 0x0, 0x100000001, 0x1, 0x14ca54188f1bdf08, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:2722 +0x22c
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).tryExecuteWriteBatch(0xc4204b8a80, 0x7fa770ec6e78, 0xc4202f4570, 0x14ca54185a7baf90, 0x0, 0x100000001, 0x1, 0x1, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:2469 +0x3e2
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).executeWriteBatch(0xc4204b8a80, 0x7fa770ec6e78, 0xc4202f4570, 0x14ca54185a7baf90, 0x0, 0x100000001, 0x1, 0x1, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:2323 +0xae
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).Send(0xc4204b8a80, 0x7fa770ec6e78, 0xc4202f4570, 0x14ca54185a7baf90, 0x0, 0x100000001, 0x1, 0x1, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:1655 +0x2a2
github.com/cockroachdb/cockroach/pkg/storage.(*Store).Send(0xc420068700, 0x7fa770ec6e78, 0xc42048ad50, 0x14ca54185a7baf90, 0x0, 0x100000001, 0x1, 0x1, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:2553 +0x6b1
github.com/cockroachdb/cockroach/pkg/storage.(*testSender).Send(0xc42052c010, 0x7fa770ec2b80, 0xc420014e40, 0x0, 0x0, 0x100000001, 0x1, 0x1, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store_test.go:105 +0x2a2
github.com/cockroachdb/cockroach/pkg/internal/client.(*DB).send(0xc420218160, 0x7fa770ec2b80, 0xc420014e40, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/internal/client/db.go:553 +0x20a
github.com/cockroachdb/cockroach/pkg/internal/client.(*DB).(github.com/cockroachdb/cockroach/pkg/internal/client.send)-fm(0x7fa770ec2b80, 0xc420014e40, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/internal/client/db.go:489 +0xa4
github.com/cockroachdb/cockroach/pkg/internal/client.sendAndFill(0x7fa770ec2b80, 0xc420014e40, 0xc4209ddb38, 0xc4206e8000, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/internal/client/db.go:461 +0x156
github.com/cockroachdb/cockroach/pkg/internal/client.(*DB).Run(0xc420218160, 0x7fa770ec2b80, 0xc420014e40, 0xc4206e8000, 0x1, 0xc4202f4480)
	/go/src/github.com/cockroachdb/cockroach/pkg/internal/client/db.go:489 +0xf8
github.com/cockroachdb/cockroach/pkg/storage.(*intentResolver).processIntents(0xc42048a3c0, 0x7fa770ec2b80, 0xc420014e40, 0xc4204b8a80, 0x3263be0, 0xc42086d7a0, 0xc42006ddd0, 0x1, 0x1, 0x14ca54185a6b8e73, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/intent_resolver.go:379 +0xc62
github.com/cockroachdb/cockroach/pkg/storage.(*intentResolver).processIntentsAsync.func1(0x7fa770ec2b80, 0xc420014e40)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/intent_resolver.go:269 +0xfd
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunLimitedAsyncTask.func1(0xc420674000, 0x7fa770ec2b80, 0xc420014e40, 0x235f60f, 0x2a, 0xc420614540, 0x0, 0x0, 0xc4203511d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:327 +0x11e
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunLimitedAsyncTask
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:328 +0x242

goroutine 1 [chan receive]:
testing.(*T).Run(0xc420067110, 0x234a2e8, 0x21, 0x23f6888, 0xc420683901)
	/usr/local/go/src/testing/testing.go:698 +0x582
testing.runTests.func1(0xc420067110)
	/usr/local/go/src/testing/testing.go:882 +0xab
testing.tRunner(0xc420067110, 0xc420683ad0)
	/usr/local/go/src/testing/testing.go:657 +0x108
testing.runTests(0xc420200860, 0x323a3e0, 0x1ba, 0x1ba, 0xc420683b50)
	/usr/local/go/src/testing/testing.go:888 +0x4e1
testing.(*M).Run(0xc4205a3f28, 0x0)
	/usr/local/go/src/testing/testing.go:822 +0x1c4
github.com/cockroachdb/cockroach/pkg/storage_test.TestMain(0xc420683f28)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/main_test.go:59 +0x308
main.main()
	github.com/cockroachdb/cockroach/pkg/storage/_test/_testmain.go:938 +0x210

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2197 +0x1

goroutine 5 [chan receive]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).flushDaemon(0x36d85c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1057 +0x85
created by github.com/cockroachdb/cockroach/pkg/util/log.init.1
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:603 +0x13a

goroutine 14 [syscall]:
os/signal.signal_recv(0x6d0181)
	/usr/local/go/src/runtime/sigqueue.go:116 +0x104
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:22 +0x30
created by os/signal.init.1
	/usr/local/go/src/os/signal/signal_unix.go:28 +0x4f

goroutine 11948 [runnable]:
internal/race.Enable()
	/usr/local/go/src/internal/race/race.go:32 +0x44
sync.(*WaitGroup).Add(0xc4201bed60, 0xffffffffffffffff)
	/usr/local/go/src/sync/waitgroup.go:96 +0x145
sync.(*WaitGroup).Done(0xc4201bed60)
	/usr/local/go/src/sync/waitgroup.go:100 +0x42
github.com/cockroachdb/cockroach/pkg/storage.(*intentResolver).resolveIntents.func2(0x7fa76fc1f0c8, 0xc4203808c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/intent_resolver.go:462 +0x11c
github.com/cockroachdb/cockroach/pkg/storage.(*intentResolver).resolveIntents.func3(0x7fa770ec6e78, 0xc420351290)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/intent_resolver.go:467 +0x51
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunLimitedAsyncTask.func1(0xc420674000, 0x7fa770ec6e78, 0xc420351290, 0x235a74c, 0x28, 0xc420614540, 0x3265ce0, 0xc42038a010, 0xc4201f28a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:327 +0x11e
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunLimitedAsyncTask
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:328 +0x242

goroutine 11886 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc40000003c)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc42037ae10)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202b8050, 0xc420674000, 0xc4202006a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11963 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Store).startGossip.func4(0x7fa770ec6e78, 0xc4203c8fc0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:1361 +0x64e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202b81a0, 0xc420674000, 0xc420406190)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11933 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000029)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc42037a5d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202115d0, 0xc420674000, 0xc4202191e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11930 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000026)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc42037a4e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211590, 0xc420674000, 0xc420219160)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11932 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000028)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc4209e02a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202115c0, 0xc420674000, 0xc4202191c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11908 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000011)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc420323200)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211430, 0xc420674000, 0xc420218d60)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11885 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000038)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc42037acc0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202b8030, 0xc420674000, 0xc420200680)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11956 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc40000003f)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc42037b110)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202b80c0, 0xc420674000, 0xc420200920)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11882 [runnable]:
sync.runtime_SemacquireMutex(0xc4200ef584)
	/usr/local/go/src/runtime/sema.go:62 +0x34
sync.(*Mutex).Lock(0xc4200ef580)
	/usr/local/go/src/sync/mutex.go:87 +0xda
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc4200ef580)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:34 +0x3b
github.com/cockroachdb/cockroach/pkg/util/caller.(*CallResolver).Lookup(0xc4200ef580, 0x4, 0x2f8167b, 0x41, 0x1d9, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/caller/resolver.go:113 +0x192
github.com/cockroachdb/cockroach/pkg/util/caller.Lookup(0x3, 0x30, 0x36d9020, 0x7fa771010e10, 0x0, 0xc4209cd658)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/caller/resolver.go:89 +0x5c
github.com/cockroachdb/cockroach/pkg/util/log.addStructured(0x7fa770ec2b80, 0xc420014e40, 0xc400000001, 0x2, 0x233c3b3, 0x19, 0xc4209cd8a0, 0x1, 0x1)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/structured.go:142 +0x5c
github.com/cockroachdb/cockroach/pkg/util/log.logDepth(0x7fa770ec2b80, 0xc420014e40, 0x1, 0x1, 0x233c3b3, 0x19, 0xc4209cd8a0, 0x1, 0x1)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/log.go:68 +0x9a
github.com/cockroachdb/cockroach/pkg/util/log.Infof(0x7fa770ec2b80, 0xc420014e40, 0x233c3b3, 0x19, 0xc4209cd8a0, 0x1, 0x1)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/log.go:86 +0x90
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Quiesce(0xc420674000, 0x7fa770ec2b80, 0xc420014e40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:473 +0x234
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc420674000, 0x7fa770ec2b80, 0xc420014e40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:418 +0x292
github.com/cockroachdb/cockroach/pkg/storage.TestEndTransactionDirectGCFailure(0xc42099c1a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica_test.go:4007 +0x5a2
testing.tRunner(0xc42099c1a0, 0x23f6888)
	/usr/local/go/src/testing/testing.go:657 +0x108
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:697 +0x544

goroutine 11928 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000024)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc4209e0270)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211570, 0xc420674000, 0xc420219100)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11902 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc40000000a)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc42048b2f0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202113d0, 0xc420674000, 0xc420218ca0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11954 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc40000003e)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc42037afc0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202b80a0, 0xc420674000, 0xc4202008e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11913 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000015)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc4202f4420)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211480, 0xc420674000, 0xc420218ea0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11905 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc40000000d)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc4202f4330)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211400, 0xc420674000, 0xc420218d00)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11912 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000012)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc420728000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211470, 0xc420674000, 0xc420218e60)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11892 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000040)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc4209e0030)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211320, 0xc420674000, 0xc420218b00)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11929 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000025)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc42037a480)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211580, 0xc420674000, 0xc420219140)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11961 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Store).startGossip.func4(0x7fa770ec6e78, 0xc42037b230)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:1361 +0x64e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202b8180, 0xc420674000, 0xc4204060f0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11937 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc40000002d)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc42037a720)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211630, 0xc420674000, 0xc4202192a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11900 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000009)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc420323110)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202113b0, 0xc420674000, 0xc420218c40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11893 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000041)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc4209e0060)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211330, 0xc420674000, 0xc420218b20)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11884 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000037)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc42037ac30)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202b8020, 0xc420674000, 0xc420200620)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11957 [semacquire]:
sync.runtime_Semacquire(0xc42006ce04)
	/usr/local/go/src/runtime/sema.go:47 +0x34
sync.(*WaitGroup).Wait(0xc42006cdf8)
	/usr/local/go/src/sync/waitgroup.go:131 +0xc2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Wait(0xc42006cd80, 0x7fa770ec6e78, 0xc42037b080)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:174 +0x3f
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Wait-fm(0x7fa770ec6e78, 0xc42037b080)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:3517 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202b80f0, 0xc420674000, 0xc4202b80d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11890 [chan receive]:
github.com/cockroachdb/cockroach/pkg/storage/engine.(*RocksDB).open.func2(0xc4204f2000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb.go:426 +0x64
created by github.com/cockroachdb/cockroach/pkg/storage/engine.(*RocksDB).open
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb.go:427 +0x6d1

goroutine 11887 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000039)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc42037acf0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202b8060, 0xc420674000, 0xc420200840)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11915 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000017)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc4209e0180)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202114a0, 0xc420674000, 0xc420218ee0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11970 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1(0x7fa770ec6e78, 0xc4209e04e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:505 +0x57c
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202b8710, 0xc420674000, 0xc4202011e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11883 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000036)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc42037ab70)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202b8010, 0xc420674000, 0xc420200520)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11903 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc40000000b)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc4209e0120)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202113e0, 0xc420674000, 0xc420218cc0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11898 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000046)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc420323050)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211390, 0xc420674000, 0xc420218c00)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11894 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000042)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc4209e0090)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211340, 0xc420674000, 0xc420218b40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11960 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Store).Start.func2(0x7fa770ec6e78, 0xc4203c8e70)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:1232 +0x19d
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202b8170, 0xc420674000, 0xc420200940)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11921 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc40000001d)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc42037a270)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211500, 0xc420674000, 0xc420219000)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11941 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000031)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc4209e0360)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211680, 0xc420674000, 0xc420219360)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11968 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1(0x7fa770ec6e78, 0xc4209e0480)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:505 +0x57c
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202b86f0, 0xc420674000, 0xc420201180)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11959 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Store).startCoalescedHeartbeatsLoop.func1(0x7fa770ec6e78, 0xc42037b1d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:3565 +0x1c5
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202b8150, 0xc420674000, 0xc4202b8130)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11922 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc40000001e)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc42037a2a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211510, 0xc420674000, 0xc420219020)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11974 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*replicaScanner).waitAndProcess(0xc420846780, 0x7fa770ec6e78, 0xc4203c9050, 0xed0dd3111, 0x248b3264, 0x36d6b80, 0xc420862000, 0xc420674000, 0xc4204b8a80, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scanner.go:210 +0x282
github.com/cockroachdb/cockroach/pkg/storage.(*replicaScanner).scanLoop.func1.1(0xc4204b8a80, 0x58)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scanner.go:272 +0x162
github.com/cockroachdb/cockroach/pkg/storage.(*storeReplicaVisitor).Visit(0xc42048a570, 0xc420407b80)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:358 +0x5ad
github.com/cockroachdb/cockroach/pkg/storage.(*replicaScanner).scanLoop.func1(0x7fa770ec6e78, 0xc4203c9050)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scanner.go:274 +0x3f0
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202b8750, 0xc420674000, 0xc420201280)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11967 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1(0x7fa770ec6e78, 0xc4209e0510)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:505 +0x57c
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202b86e0, 0xc420674000, 0xc420201120)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11938 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc40000002e)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc4209e0300)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211640, 0xc420674000, 0xc420219300)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11901 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000008)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc4202f4300)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202113c0, 0xc420674000, 0xc420218c80)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11972 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1(0x7fa770ec6e78, 0xc4209e0540)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:505 +0x57c
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202b8730, 0xc420674000, 0xc420201240)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11969 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1(0x7fa770ec6e78, 0xc4209e04b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:505 +0x57c
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202b8700, 0xc420674000, 0xc4202011a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11904 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc40000000c)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc420323140)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202113f0, 0xc420674000, 0xc420218ce0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11906 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc40000000e)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc4202f4360)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211410, 0xc420674000, 0xc420218d20)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11888 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc40000003a)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc42037ad20)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202b8070, 0xc420674000, 0xc420200880)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11925 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000021)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc42037a360)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211540, 0xc420674000, 0xc420219080)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11914 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000016)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc42037a090)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211490, 0xc420674000, 0xc420218ec0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11916 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000018)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc42037a120)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202114b0, 0xc420674000, 0xc420218f20)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11911 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc40000000f)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc4203231a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211460, 0xc420674000, 0xc420218e40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11971 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1(0x7fa770ec6e78, 0xc4209e0420)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:505 +0x57c
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202b8720, 0xc420674000, 0xc420201220)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11895 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000043)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc420322e40)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211350, 0xc420674000, 0xc420218b60)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11943 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000033)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc4209e03c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202116a0, 0xc420674000, 0xc4202193c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11896 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000044)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc4209e00c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420211360, 0xc420674000, 0xc420218b80)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11935 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc40000002b)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:199 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fa770ec6e78, 0xc42037a600)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:168 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202115f0, 0xc420674000, 0xc420219240)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xbb

goroutine 11899 [semacquire]:
sync.runtime_notifyListWait(0xc4201f0190, 0xc400000007)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4201f0180)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42006cd80, 0xc420674000)
	/go/src/github.com/cockroach
```
Please assign, take a look and update the issue accordingly.

--- a/pkg/storage/intent_resolver.go
+++ b/pkg/storage/intent_resolver.go
@@ -457,13 +457,14 @@ func (ir *intentResolver) resolveIntents(
 		}
 		wg.Add(1)
 		action := func() error {
-			defer wg.Done()
 			// TODO(tschottdorf): no tracing here yet.
 			return ir.store.DB().Run(ctx, b)
 		}
 		if ir.store.Stopper().RunLimitedAsyncTask(
 			ctx, "storage.intentResolve: resolving intents", ir.sem, true, /* wait */
 			func(ctx context.Context) {
+				defer wg.Done()
+
 				if err := action(); err != nil {
 					// If we have a waiting caller, pass the first non-nil
 					// error out on the channel.
@@ -476,6 +477,7 @@ func (ir *intentResolver) resolveIntents(
 					log.Warningf(ctx, "unable to resolve external intents: %s", err)
 				}
 			}) != nil {
+			wg.Done()
 			// Try async to not keep the caller waiting, but when draining
 			// just go ahead and do it synchronously. See #1684.
 			// TODO(tschottdorf): This is ripe for removal.

--- a/pkg/storage/replica_test.go
+++ b/pkg/storage/replica_test.go
@@ -3981,7 +3981,8 @@ func TestEndTransactionDirectGCFailure(t *testing.T) {
 				atomic.AddInt64(&count, 1)
 				return roachpb.NewErrorWithTxn(errors.Errorf("boom"), filterArgs.Hdr.Txn)
 			} else if filterArgs.Req.Method() == roachpb.GC {
-				t.Fatalf("unexpected GCRequest: %+v", filterArgs.Req)
+				// Can't fatal since we're on a goroutine. This'll do it.
+				t.Error(errors.Errorf("unexpected GCRequest: %+v", filterArgs.Req))
 			}
 			return nil
 		}

